### PR TITLE
Households

### PR DESCRIPTION
**Pull request by capitaladot**
This should be ready.

### CODE REVIEW

File: Household.cat [hunk 1]
```diff
@@ -0,0 +1,2401 @@
...
+      <costs>
+        <cost name=" Stratagem Points" typeId="efbf-52f7-fd08-f329" value="2.0"/>
+        <cost name=" Points" typeId="a731-e220-2d8a-41bf" value="0.0"/>
+      </costs>
+    </selectionEntry>
+    <selectionEntry id="2689-80b2-03ef-25fa" name="Plasma Mines" publicationId="975a-00f4-pubN89746" page="39" hidden="false" collective="false" import="false" type="upgrade">
+      <rules>
+        <rule id="c396-700d-a9d6-875b" name="Plasma Mines" publicationId="975a-00f4-pubN89746" page="39" hidden="false">
+          <description>Play this Stratagem immediately after an
+enemy unit finishes moving or making a turn. That unit suffers D3 Strength 10 hits. If the unit is a Titan, these will be ro its Legs. Void Shield saves cannot be made against the hits, but Ion
```
Comment (TrentBartlem) [inline Household.cat:317]: > these will be ro its Legs

*to* its legs

File: Household.cat [hunk 2]
```diff
@@ -0,0 +1,2401 @@
...
+        <cost name=" Stratagem Points" typeId="efbf-52f7-fd08-f329" value="2.0"/>
+        <cost name=" Points" typeId="a731-e220-2d8a-41bf" value="0.0"/>
+      </costs>
+    </selectionEntry>
+    <selectionEntry id="06d7-5c7b-eefe-99e3" name="Orbital Lance Strike" publicationId="975a-00f4-pubN89746" page="40" hidden="false" collective="false" import="false" type="upgrade">
+      <rules>
+        <rule id="689d-a3f2-a626-703f" name="Orbital Lance Strike" publicationId="975a-00f4-pubN89746" page="40" hidden="false">
+          <description>Once per battle, play this during
+the Strategy phase. Place the 3&quot; Blasr marker anywhere on
+the bartlefield, then scatter it D6&quot;. Any unir touched by the
```
Comment (TrentBartlem) [inline Household.cat:299]: 'unit', 'battlefield', 'blast', 'it' , '10'

File: Household.cat [hunk 3]
```diff
@@ -0,0 +1,2401 @@
...
+      </categoryLinks>
+      <costs>
+        <cost name=" Stratagem Points" typeId="efbf-52f7-fd08-f329" value="1.0"/>
+        <cost name=" Points" typeId="a731-e220-2d8a-41bf" value="0.0"/>
+      </costs>
+    </selectionEntry>
+    <selectionEntry id="0a6b-13c3-78ee-7d67" name="Decapitating Strike" publicationId="975a-00f4-pubN89746" page="40" hidden="false" collective="false" import="false" type="upgrade">
+      <rules>
+        <rule id="9aad-e84d-7e73-0eb3" name="Decapitating Strike" publicationId="975a-00f4-pubN89746" page="40" hidden="false">
+          <description>Play this at rhe end of the battle. If the
```
Comment (TrentBartlem) [inline Household.cat:229]: 'rhe'

**Comment by capitaladot** [comment]
[Household Test.zip](https://github.com/BSData/adeptus-titanicus/files/5900526/Household.Test.zip)

File: Household.cat [hunk 4]
```diff
@@ -0,0 +1,5657 @@
...
+                    </selectionEntryGroup>
+                  </selectionEntryGroups>
+                  <costs>
+                    <cost name=" Stratagem Points" typeId="efbf-52f7-fd08-f329" value="0.0"/>
+                    <cost name=" Points" typeId="a731-e220-2d8a-41bf" value="45.0"/>
+                  </costs>
+                </selectionEntry>
+                <selectionEntry id="b09f-9393-7c1b-10f9" name="Questoris Knight Lord Magaera" hidden="false" collective="false" import="true" type="model">
+                  <constraints>
+                    <constraint field="sel
```
Comment (TrentBartlem) [inline Household.cat:5545]: is this description missing a word?

Comment (TrentBartlem) [inline Household.cat:5499]: missing a word at the end?

Comment (TrentBartlem) [inline Household.cat:5411]: spaces

Comment (capitaladot) [reply Household.cat:5545]: Nope!

Comment (capitaladot) [reply Household.cat:5499]: Same line as above? Maybe your marker ended up off?

Comment (capitaladot) [reply Household.cat:5411]: I don't see any spaces or missing need?

Comment (TrentBartlem) [inline Household.cat:4413]: 'banle'

Comment (TrentBartlem) [inline Household.cat:4385]: 'the Senechal. is'

Comment (TrentBartlem) [inline Household.cat:4353]: 'higher than their'

Comment (TrentBartlem) [inline Household.cat:4213]: 'results of an I'

Comment (TrentBartlem) [inline Household.cat:4199]: 'an.d'

Comment (TrentBartlem) [inline Household.cat:4184]: 'add I '. probably worth search and replacing

Comment (TrentBartlem) [inline Household.cat:4169]: 'deduct I '

Comment (TrentBartlem) [inline Household.cat:4041]: 'set char unit'. Also missing text

Comment (TrentBartlem) [inline Household.cat:4004]: ' on a r, '

Comment (TrentBartlem) [inline Household.cat:3947]: 'where ir ', 'strength io'

Comment (TrentBartlem) [inline Household.cat:3919]: 'scarcer it', 'Dio'

Comment (TrentBartlem) [inline Household.cat:3905]: Strucrurally

**Review by TrentBartlem — CHANGES_REQUESTED** [review]
typos etc

**Comment by capitaladot** [comment]
> still DIO instead of D10

Got it this time.

**Comment by Deviltree** [comment]
I just pulled this build down from the normal battle scribe data update and it's completely broken.

**Pull request merged**